extensions/crypto/lib.deno_crypto.d.ts

```diff
@@ -49,9 +128,32 @@ interface SubtleCrypto {
   ): Promise<ArrayBuffer>;
 }
 
+declare interface Crypto {
+  readonly subtle: SubtleCrypto;
+  getRandomValues<
+    T extends
+      | Int8Array
+      | Int16Array
+      | Int32Array
+      | Uint8Array
+      | Uint16Array
+      | Uint32Array
+      | Uint8ClampedArray
+      | Float32Array
+      | Float64Array
+      | DataView
+      | null,
+  >(
+    array: T,
+  ): T;
+  randomUUID(): string;
```

Comment:
Is randomUUID implemented? I've never seen it in any specification nor I didn't spot an implementation.

Reply:
It is implemented and specified: https://wicg.github.io/uuid/. It is shipping in Chrome 92 and has already shipped in Node.

Reply:
Oh, I hadn't heard of that draft spec yet, thanks!